taxi/tests/test_forms.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+from django.test import Client
+from django.urls import reverse
+
+from taxi.models import Driver, Manufacturer, Car
+
+
+@pytest.fixture
+def authorize_user():
+    user = Driver.objects.create_user(
+        username="Test_name",
+        password="Test_1234",
+        license_number="ABC12345"
+    )
+    client = Client()
+    client.force_login(user)
+    return client
+
+
+@pytest.mark.django_db
+def test_correct_searching_manufacturer(authorize_user):
+    Manufacturer.objects.bulk_create([
+        Manufacturer(name="BB_test", country="country"),
+        Manufacturer(name="AA_test", country="country")
+    ])
+    url = reverse("taxi:manufacturer-list") + "?name=AA"
+    res = authorize_user.get(url)
+    assert "AA_test" in res.content.decode()
+    assert "BB_test" not in res.content.decode()
+
+
+@pytest.mark.django_db
+def test_correct_searching_car(authorize_user):
+    manufacturer = Manufacturer.objects.create(name="test", country="country")
+    Car.objects.bulk_create([
+        Car(model="BB_test", manufacturer=manufacturer),
+        Car(model="AA_test", manufacturer=manufacturer)
+    ])
+    url = reverse("taxi:car-list") + "?model=AA"
+    res = authorize_user.get(url)
+    assert "AA_test" in res.content.decode()
+    assert "BB_test" not in res.content.decode()
+
+
+@pytest.mark.django_db
+def test_correct_searching_driver(authorize_user):
+    Driver.objects.create_user(
+        username="AA_test",
+        password="Test_1234",
+        license_number="AAA11111"
+    )
+    Driver.objects.create_user(
+        username="BB_test",
+        password="Test_1234",
+        license_number="AAA11112"
+    )
+
```

Comment:
Consider adding an assertion for the response status code to ensure the request is successful.